scale_build/config.py

```diff
@@ -0,0 +1,21 @@
+import os
+import time
+
+from datetime import datetime
+
+
+BUILD_TIME = int(time.time())
+BUILD_TIME_OBJ = datetime.fromtimestamp(BUILD_TIME)
+BUILDER_DIR = os.getenv('BUILDER_DIR', './')
+BRANCH_OVERRIDES = {k[:-(len('_OVERRIDE'))]: v for k, v in os.environ.items() if k.endswith('_OVERRIDE')}
+CODE_NAME = os.getenv('CODENAME', 'Angelfish')
+PARALLEL_BUILD = int(os.getenv('PARALLEL_BUILDS', max(os.cpu_count(), 8) / 4))
```

Comment:
Is this to ensure we always have parallel builds even on systems with 4 CPUs?

Reply:
Yes this was to ensure that with machine less then 8 cpus, we have parallel builds by default unless overridden

Reply:
Why? Parallel build testing purposes?

Reply:
Well from my tests, i see that when machines with 8 cpus can still see performance gains with 2 threads whereas when we increase this number, we should be dividing by 4 as some packages are already consuming all cores like kernel/openzfs.